quickwit-metastore/src/metastore/postgresql_metastore.rs

```diff
@@ -330,8 +333,21 @@ impl PostgresqlMetastore {
 
         if let Some(tags) = tags_opt {
             if !tags.is_empty() {
-                select_statement =
-                    select_statement.filter(schema::splits::dsl::tags.overlaps_with(tags));
+                // Rebuild split tags by escaping values & taking into account
```

Comment:
One question here: do we need to add this piece of code for each metastore implementation?

Reply:
Ok nevermind, I saw the common function `match_tags_filter ` so forget my comment.

Reply:
This is a good point will attempt to lift this, let's discuss options once I gather them.